pubsub/spec/test_test.rb

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "spec_helper"
+
+RSpec.describe "Pub/Sub test test" do
```

Comment:
I just wanted to make sure that the travis setup actually worked with more than one directory. I'll submit the PR with the pub sub sample I have right away.